### DISCUSSION TITLE
How to replace Dosbox by Dosemu2

### DISCUSSION THREAD

**Discussion by bodhi-baum** [dosemu2/dosemu2#1560] [General]
I would like to replace Dosbox by Dosemu2 but I don´t know how.
That may not be a real issue but could be put into the wiki, if it becomes a good howto.

**Comment by stsp** [comment]
There is no such howto, because
nobody knows why and where do
you want to replace dosbox with
dosemu2.
You can install them both, and use
separately to run various progs.
Some progs work better with dosbox,
some with dosemu2.

**Reply by stsp** [reply]
Use `dosbox ./prog.exe`
or `dosemu -K ./prog.exe`
and see what suits best.
